winex.js

```diff
@@ -189,7 +189,7 @@ factory = function(winstonLogger, classMeta, opts) {
           }
           level = "info";
           if ((400 <= (_ref = res.statusCode) && _ref < 500)) {
-            level = "warn";
+            level = "warning";
```

Comment:
The levels that are actually attached to Log prototype as methods come from winstonTransport, so hard-coding info, warn/warning, and error in this section of code is making assumptions about what winstonTransport's levels are. This has tripped me up in the past, so I wanted to make a note of it.